### PR TITLE
[CRAW-190] Add additional mark with inject time info.

### PR DESCRIPTION
**Pull request by vanchpuck**

### CODE REVIEW

File: src/java/org/apache/nutch/storage/Mark.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
 public enum Mark {
   INJECT_MARK("_injmrk_"), GENERATE_MARK("_gnmrk_"), FETCH_MARK("_ftcmrk_"),
   PARSE_MARK("__prsmrk__"), UPDATEDB_MARK("_updmrk_"), INDEX_MARK("_idxmrk_"),
-  SLOW_MARK("_slwmrk_"), INPUT_MARK("_inptmrk_");
+  SLOW_MARK("_slwmrk_"), INPUT_MARK("_inptmrk_"), INJECT_TIME_MARK("_injtmmrk_");
```
Comment (SergeyWeiss) [inline src/java/org/apache/nutch/storage/Mark.java]: We keep on adding marks that do not belong here. It kind of feels wrong

**Comment by kgusakov** [comment]
MRUnit test?

**Comment by kgusakov** [comment]
:+1:

File: src/java/org/apache/nutch/storage/Mark.java (hunk 1 above)
Comment (vanchpuck) [inline src/java/org/apache/nutch/storage/Mark.java]: Alternatives?

Comment (SergeyWeiss) [inline src/java/org/apache/nutch/storage/Mark.java]: We may investigate why Kirill failed to store such data in metadata

**Pull request merged**